.github/workflows/doctest.yaml

```
@@ -13,10 +13,9 @@ jobs:
                'doc/starterkit/k4MarlinWrapperCLIC/CEDViaWrapper']
     steps:
     - uses: actions/checkout@v3
-    - uses: cvmfs-contrib/github-action-cvmfs@v3.1
     - name: Start container
       run: |
-        docker run -it --name CI_container -v ${GITHUB_WORKSPACE}:/Package -v /cvmfs:/cvmfs:shared -d ghcr.io/aidasoft/centos7:latest /bin/bash
+        docker run -it --name CI_container --privileged -v ${GITHUB_WORKSPACE}:/Package -v /cvmfs:/cvmfs:shared -d ghcr.io/key4hep/key4hep-images/alma9-cvmfs /bin/bash
     - name: Compile
       run: |
         docker exec CI_container /bin/bash -c ' cd Package;/
```

This also doesn't look right - has been broken for a while apparently :see_no_evil: 

```suggestion
        docker exec CI_container /bin/bash -c ' cd Package;\
```

---

It was broken for a while because we didn't build the necessary jupytext any longer. But that should have been fixed now.

---

Although this bit has definitely been working in the past

---

Yes, I guess the newline escape isn't needed, then it would just try to execute the `/`, fail, and continue.